### PR TITLE
Feature: Javascript client

### PR DESCRIPTION
**Pull request by zouzias**
This PR provides provides an implementation of a simple nodejs application that restifies the GSIS SOAP webservice.

The web application has also been deployed on Heroku, see https://greek-vat-data.herokuapp.com. Deployment code is here: https://github.com/zouzias/greek-vat-data-heroku

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+'use strict';
```
Comment (dspinellis) [inline js/app.js:1]: Please add a short description and the license boilerplate.

**Comment by dspinellis** [comment]
That's great!  Please also update the top-level README file.

File: js/app.js (hunk 1 above)
Comment (zouzias) [inline js/app.js:1]: I forgot this one ;-) Added in 391c3ec

**Comment by zouzias** [comment]
Updated top-level README in 4040b79

**Pull request merged**